mlflow/langchain/utils/chat.py

```diff
@@ -209,9 +209,9 @@ def _gen_converted_chunk(message_content, message_id, finish_reason):
         )
 
         if IS_PYDANTIC_V2_OR_NEWER:
-            return transformed_response.model_dump(mode="json")
+            return transformed_response.model_dump(mode="json", exclude_unset=True)
         else:
-            return json.loads(transformed_response.json())
+            return json.loads(transformed_response.json(exclude_unset=True))
```

Comment:
is the change for fixing error of missing "tool_calls" field (None value) in ChatChoiceDelta ?